jdi-light/src/main/java/com/epam/jdi/light/elements/common/UIElement.java

```diff
@@ -841,7 +841,8 @@ protected boolean selected() {
         if (getWebElement().isSelected())
             return true;
         return hasClass("checked") || hasClass("active") ||
-            hasClass("selected") || attr("checked").equals("true");
+                hasClass("selected") || attr("checked").equals("true") ||
+                attr("class").contains("marked");
     }
     protected boolean enabled() {
         if (hasClass("active"))
```

Comment:
We should definitely discuss it with Vladimir as it is key basic element (I mean UIElement)

Reply:
fixed